src/serialchemy/_tests/test_nested_fields.py

```diff
@@ -60,7 +60,7 @@ def test_deserialize_with_custom_serializer(db_session, data_regression):
         "lastname": "Doe",
         "company_id": 5,
         "admission": "2004-06-01T00:00:00",
-        "address": {"number": "245", "street": "6 Av", "zip": "88088-000"},
+        "address": {"id": 1, "number": "245", "street": "6 Av", "zip": "88088-000"},
```

Comment:
Would be nice to have a test that fails without that changes, if possible. Instead of editing this one.